### PR TITLE
BETA: Register vo.is-a.dev

### PR DESCRIPTION
**Pull request by YoruAkio**
Added `vo.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/vo.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "YoruAkio",
+        "email": "yoruakio@proton.me"
+    },
+    "record": {
+        "CNAME": "voids.pages.dev"
```
Comment (DEV-DIBSTER) [inline domains/vo.json:7]: Unable to access CNAME.

File: domains/v0.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "YoruAkio",
+        "email": "yoruakio@proton.me"
+    },
+    "record": {
+        "CNAME": "vo-kio.pages.dev"
```
Comment (DEV-DIBSTER) [inline domains/v0.json:7]: Little content displayed.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by YoruAkio — COMMENTED** [review]
Please review this changes!

**Comment by YoruAkio** [comment]
> # 🔍 ReviewMate Analysis
> File: [domains/v0.json](https://github.com/is-a-dev/register/blob/7ef10705002d44696100e15c45688f751105d164/domains%2Fv0.json) Content URL: http://vo-kio.pages.dev Description: N/A Repository: N/A
> 
> ## 📸 Screenshot
> Screenshot of domains/v0.json

**Fix**: Added more content to displayed!

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**